release/ray_release/reporter/db.py

```diff
@@ -24,6 +26,8 @@ def report_result(self, test: Test, result: Result):
             "_table": "release_test_result",
             "report_timestamp_ms": int(time.time() * 1000),
             "status": result.status or "",
+            "branch": os.environ.get("BUILDKITE_BRANCH", ""),
+            "commit": os.environ.get("BUILDKITE_COMMIT", ""),
```

Comment:
not sure if there are cases where `BUILDKITE_COMMIT` can be just `HEAD` anymore?

Reply:
probably not; as commit is used as the index to get rayproject/ray image